responsive-web-design/Build-product-landing-page/creative.css

```diff
@@ -0,0 +1,276 @@
+    width: 12vw;
+    color: #030303;
+}
+
+.grid .right-element{
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    height: 109px;
+    padding: 5px;
+    font-weight: 700;
+} 
+
+#how-it-works{
+    margin-top: 43px;
+    display: flex;
+    justify-content: center;
+    width: 100%;
+}
+
+#video{
+    width: 526px;
+    height: 294px;
+}
+
+#pricing{
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin-top: 60px
+}
+
+.product{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    width: calc(100% / 3);
+    margin: 10px;
+    border: 1px solid #1000ff;
+    background-color: #f11a1ae6;
+    border-radius: 3px;
+}
+
+.product  .level{
+    background-color: rgb(8, 8, 8);
+    color: white;
+    padding: 15px 0;
+    width: 100%;
+    text-transform: uppercase;
+    font-weight: 800;
+}
+
+.product  h2{
+    margin-top: 15px;
+}
+
+.product  ol{
+    margin: 15px 0;
+    font-weight: 600;
+    list-style: none;
+}
+
+.product  ol  li{
+    padding: 4px 0;
+    text-align: justify;
+}
+
+.product  button {
+    border: 0;
+    margin: 15px 0;
+    background-color: #eee;
+    font-weight: 400;
+}
+
+.btn{
+    padding: 0 20px;
+    height: 40px;
+    font-size: 1em;
+    font-weight: 900;
+    text-transform: uppercase;
+    border-radius: 3px;
+    cursor: pointer;
+}
+
+footer {
+    background: cornsilk;
+    padding: 3px 20px 8px 20px;
+    margin: 27px 60px 0px 60px;
+    border-radius: 5px;
+    background-color: #de2222;
+}
+
+footer ul{
+    display: flex;
+    justify-content: flex-end;
+    padding: 0px;
+    list-style: none;
+}
+
+footer ul  li {
+    padding: 10px;
+}
+
+footer  span {
+    margin-top: 0px;
+    display: flex;
+    justify-content: flex-end;
+    font-size: 0.9em;
+    color: rgb(36, 35, 35);
+    padding: 0px 10px;
+}
+
+@media (max-width: 767px) {
+.link-buttons{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+ }
+.container{
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+}
+
+#nav-bar{
+    display: flex;
+    justify-content: flex-end;
+    width: inherit;
+}
+    
+#nav-bar li{
+    text-align:  center;
+    width: 100%;
+    margin: auto;
+}
+
+#header nav ul li {
+    display: flex;
+    padding: 3px;       
+    font-weight: 100;
+}
+
+#header {
+    display: flex;
+    flex-flow: wrap;
+    width: 100%;
+}
+
+#form-elements{
+    display: flex;
+    justify-content: flex-end;
+    text-align: -webkit-center;
+    margin-bottom: 40px;
+    height: 338px;
+}
+
+.logo {
+    display: flex;
+    justify-content: center;
+    width: 100%;
+}
+
+#features {
+    margin-top: 0px;
+}
+
+#pricing{    
+   display: flex;        
+   flex-direction: column;      
+   align-items: center;
+}
+.product{
+   width: 90%;
+}
+
+#features .icon {
+   display: none;
+}
+
+#features .grid {
+    margin-bottom: 70px;
+    }
+}
```

Comment:
We have the indentation wrong yet, please fix these lines: https://github.com/andcoding/freecodecamp-courses/blob/8ff1e4a39837fdee9584390af70320357cf18876/responsive-web-design/Build-product-landing-page/creative.css#L203-L274

Using this code as an example:

```css
@media (max-width: 767px) {
    .link-buttons{
        display: flex;
        flex-direction: column;
        align-items: center;
        width: 100%;
     }
    .container{
        display: flex;
        flex-direction: column;
        text-align: center;
    }

    #nav-bar{
        display: flex;
        justify-content: flex-end;
        width: inherit;
    }
        
    #nav-bar li{
        text-align:  center;
        width: 100%;
        margin: auto;
    }

    #header nav ul li {
        display: flex;
        padding: 3px;       
        font-weight: 100;
    }

    #header {
        display: flex;
        flex-flow: wrap;
        width: 100%;
    }

    #form-elements{
        display: flex;
        justify-content: flex-end;
        text-align: -webkit-center;
        margin-bottom: 40px;
        height: 338px;
    }

    .logo {
        display: flex;
        justify-content: center;
        width: 100%;
    }

    #features {
        margin-top: 0px;
    }

    #pricing{    
       display: flex;        
       flex-direction: column;      
       align-items: center;
    }
    .product{
       width: 90%;
    }

    #features .icon {
       display: none;
    }

    #features .grid {
        margin-bottom: 70px;
    }
}
```